daemon/daemon.go

```diff
@@ -980,6 +985,10 @@ func initNetworkController(config *Config) (libnetwork.NetworkController, error)
 	return controller, nil
 }
 
+func (daemon *Daemon) NetworkApiRouter() func(w http.ResponseWriter, req *http.Request) {
```

Comment:
Maybe instead of importing http here we can define this as a type on api/server and use that instead?

I don't like that we're importing http here in daemon.